src/dataconnect/provisionCloudSql.ts

```diff
@@ -106,6 +111,7 @@ export async function provisionCloudSql(args: {
     } else {
       // Skip it if the database is not accessible.
       // Possible that the CSQL instance is in the middle of something.
+      logger.debug(`Unexpected error from CloudSQL: ${err}`);
```

Comment:
Do we want to keep this debug log?

Reply:
Yep, it might be useful in cases users hit this in the wild.